packages/shared/src/lib/query.ts

```diff
@@ -35,6 +35,10 @@ export enum OtherFeedPage {
   SourceBestDiscussed = 'sources[source]/best-discussed',
   TagsMostUpvoted = 'tags[tag]/most-upvoted',
   TagsBestDiscussed = 'tags[tag]/best-discussed',
+  Explore = 'posts',
+  ExploreLatest = 'postslatest',
+  ExploreDiscussed = 'postsdiscussed',
+  ExploreUpvoted = 'postsupvoted',
```

Comment:
Why did they end up getting moved to `OtherFeedPage`? The reason they are part of `SharedFeedPage` is so we can easily understand they are shared across both webapp and extension which is true for this case.